docs/web/usecases-implement-component-participate-app-config.mdx

```diff
@@ -378,7 +381,7 @@ import Collection from "esri/core/Collection";
 
```

Comment:
I notice in the `model.newsItems.map` above we don't call `toArray()` like we do in other examples.

Reply:
Good catch! Fixed